src/python/ensembl/io/genomio/manifest/check_integrity.py

```diff
@@ -340,8 +340,19 @@ def _retrieve_gff_gene_lengths(
             gene_id = gene_id.replace("gene:", "")
         genes[gene_id] = abs(feat.location.end - feat.location.start)
         # Get CDS id and length
+        protein_transcripts = {
+            "mRNA",
+            "pseudogenic_transcript",
+        }
+        ig_transcripts = {
+            "IG_V_gene",
+            "IG_C_gene",
+            "TR_C_gene",
+            "TR_V_gene",
+        }
+        cds_transcripts = protein_transcripts | ig_transcripts
```

Comment:
```suggestion
        cds_transcripts = protein_transcripts.union(ig_transcripts)
```
Just because I find it more readable to people less familiar with sets in Python. But it can stay.